src/resolve/resolved-dns-scope.c

```diff
@@ -39,6 +39,8 @@
 #define MULTICAST_RESEND_TIMEOUT_MIN_USEC (100 * USEC_PER_MSEC)
 #define MULTICAST_RESEND_TIMEOUT_MAX_USEC (1 * USEC_PER_SEC)
 
+static const char *dnssd_services_keyname = "_services._dns-sd._udp.local";
```

Comment:
why this static variable? why not specify this string literally where it is needed (or use `#define` if you must)